src/control/client/connect.go

```diff
@@ -78,6 +78,7 @@ type Connect interface {
 	StoragePrepare(*ctlpb.StoragePrepareReq) ResultMap
 	SystemMemberQuery() (common.SystemMembers, error)
 	SystemStop() (common.SystemMemberResults, error)
+	LeaderQuery(string) (string, []string, error)
```

Comment:
Don't want to go with request/response structures? A response structure to package the leader and replicas together would work better, I think, than returning the bare strings.

Reply:
Yeah... You're right. I'll fix that.